<?xml version="1.0"?>

<launch>
  <param name="/use_sim_time" value="false" />

  <!-- This launches a mongo db server -->
  <include file="$(find flor_walk_monitor)/launch/mongodb.launch"/>

  <!-- This launches the walk monitor vis node -->
  <node name="flor_walk_monitor_vis" pkg="flor_walk_monitor" type="walk_monitor_vis_node" output="screen" >
    <rosparam file="$(find flor_walk_monitor)/config/walk_monitor_params.yaml" command="load" />
    <rosparam file="$(find flor_footstep_planner)/config/atlas_params.yaml" command="load" />
    <param name="log_folder" value="$(find flor_walk_monitor)/results/" />
  </node>
</launch>
